packages/dd-trace/test/setup/mocha.js

```diff
@@ -75,7 +75,7 @@ function withNamingSchema (spanProducerFn, expectedOpName, expectedServiceName,
             })
             .then(done)
             .catch(done)
-          spanProducerFn()
+          spanProducerFn(done)
```

Comment:
This looks like a bug? When the `.then(done)` above calls, it'll execute the `done()` function. However, when `spanProducerFn(done)` calls, it will also run `done()`. Running this function twice results in a test failure. Was it added accidentally?

Reply:
Unless it's explicitly used for the purpose of failing tests and should always have a truthy argument?